docs/buildah-push.md

```diff
@@ -45,6 +45,10 @@ If the transport part of DESTINATION is omitted, "docker://" is assumed.
 
 ## OPTIONS
 
+**--all-tags, -a**
+
+All tagged images in the repository will be pushed, not just `:latest`.  A tag can not be included in the imageID or DESTINATION fields.
```

Comment:
`:latest` is not the only tag one can pull, it’s only the default if the user specifies no tag and no digest.